src/components/ColorPicker/ColorInput.tsx

```diff
@@ -36,30 +42,35 @@ export const ColorInput: FC<ColorInputProps> = ({
         <div
             {...focusProps}
             className={merge([
-                "tw-flex tw-items-center tw-h-9 tw-gap-1 tw-px-3 md:tw-px-1 tw-border tw-rounded tw-text-s tw-font-sans tw-relative tw-bg-white dark:tw-bg-transparent focus-within:tw-border-black-90 tw-border-black-20",
+                "tw-flex tw-items-center tw-h-9 tw-px-3 tw-border tw-rounded tw-text-s tw-font-sans tw-relative tw-bg-white dark:tw-bg-transparent focus-within:tw-border-black-90 tw-border-black-20",
                 isFocusVisible && FOCUS_STYLE,
             ])}
             data-test-id="color-input"
         >
-            {decorator && (
-                <div className="tw-flex tw-items-center tw-justify-center md:tw-pl-0.5 tw-text-black-80">
-                    {decorator}
-                </div>
-            )}
-            <input
-                id={useMemoizedId()}
-                ref={inputElement}
-                className="tw-grow tw-border-none tw-outline-none tw-bg-transparent tw-hide-input-arrows tw-text-black tw-placeholder-black-60 dark:tw-text-white"
-                onClick={() => inputElement.current?.focus()}
-                onKeyDown={handleKeyDown}
-                onChange={(event) => onChange && onChange(event.currentTarget.value)}
-                onBlur={onBlur}
-                value={value}
-                type={type}
-                min={min}
-                max={max}
-                size={size}
-            />
+            <div
+                className={merge([
+                    "tw-flex",
+                    decoratorPosition === DecoratorPosition.Right ? "tw-flex-row-reverse" : "tw-gap-1",
```

Comment:
The thing with the gap is, that the input is currently left aligned. This looks weird if you type in a value < 100, so I'd suggest you right-align the text if the decorator is on the right (see comment below)
```suggestion
                    "tw-flex tw-gap-1",
                    decoratorPosition === DecoratorPosition.Right && "tw-flex-row-reverse",
```

Reply:
good idea, thanks. changed it